internal/fs/kernel_list_cache_test.go

```diff
@@ -120,6 +122,12 @@ func (t *KernelListCacheTestWithPositiveTtl) SetupSuite() {
 }
 
 func TestKernelListCacheTestWithPositiveTtlSuite(t *testing.T) {
+	kernelVersion := operations.KernelVersion(t)
+	matched, err := regexp.MatchString(UnsupportedKernelVersion, kernelVersion)
```

Comment:
If possible, we have a utils class we could have a top level function, isKernelListCacheEnabled() and then it makes the tests more readable and one place to update accordingly if other linux versions come or needs updation.

You can also add a todo in the code if we want to unblock currently and do it later.

Reply:
I think the best place to keep this would be in fstest setupTestSuite but currently it does not support taking Testing.T. I have created a TODO bug to fix it once we fully migrate to stretchr testify.